astropy/table/serialize.py

```diff
@@ -37,6 +37,7 @@
     'astropy.coordinates.earth.EarthLocation',
     'astropy.coordinates.sky_coordinate.SkyCoord',
     'astropy.table.table.NdarrayMixin',
```

Comment:
Might it make sense to move the location here to where it actually is? (Maybe only possible if one does not define `__module__` - which still feels like a hack best avoided...)